plugins/casutg/src/main/java/com/blazemeter/jmeter/threads/AdditionalFieldsPanel.java

```diff
@@ -28,6 +29,7 @@ public AdditionalFieldsPanel(boolean showConcurrencyLimit) {
         add("Time Unit: ", groupPanel);
 
         add("Thread Iterations Limit: ", iterations);
+        add("Same user on each iteration", sameUserBox);
```

Comment:
Please use the same work capitalization style as on other labels

Reply:
Good catch, I copied the original text from JMeter and didn't realize it. Thanks.